lib/routes/token.js

```diff
@@ -127,7 +138,10 @@ module.exports = {
     // we don't use, such as `response_type`, or something else. Instead
     // of giving an error here, we can just ignore them.
     payload: function validatePayload(value, options, next) {
-      return Joi.validate(value, PAYLOAD_SCHEMA, { stripUnknown: true }, next);
+      return Joi.validate(value, PAYLOAD_SCHEMA, { stripUnknown: true }, function(err, value) {
+        console.log(err);
+        next(err, value)
```

Comment:
I'm surprised we don't have this right now, the detailed validation fails just kind of get dropped and are not logged it seems.